docs/mediapipe.md

```diff
@@ -132,53 +75,91 @@ With KServe gRPC API you are also able to push side input packets into graph. In
+When the client is sending in the gRPC request the input as an numpy array, it will be deserialized on the Model Server side to the format specified in the graph.
+For example when the graph has the input type IMAGE, the gRPC client could send the input data with the shape `(300, 300, 3)` and precision INT8. It would not be allowed to send the data in the shape for example `(1,300,300,1)` as that would be incorrect layout and the number of dimensions.
 
-Review also an example in [object detection demo](../demos/mediapipe/object_detection/README.md)
+When the input graph would be set as `OVTENSOR`, an arbitrary shape and precisions on the input would be allowed. It will be converted to `OV::Tensor` object and passed to the graph. For example input with shape `(1,3,300,300) FP32` assuming that format would be accepted by the graph calculators.
 
-## Configuration files <a name="configuration-files"></a>
-MediaPipe servables configuration is to be placed in the same json file like the 
-[models config file](starting_server.md).
-While models are defined in section `model_config_list`, graphs are configured in
-the `mediapipe_config_list` section. 
+Check the code snippets for [gRPC unary](https://docs.openvino.ai/2023.2/ovms_docs_clients_kfs.html#request-prediction-on-a-numpy-array) calls and [gRPC streaming](https://docs.openvino.ai/2023.2/ovms_docs_clients_kfs.html#request-streaming-prediction).
 
-Basic graph section template is depicted below:
+There is also an option to avoid any data conversions in the serialization and deserialization by the OpenVINO Model Server. When the input stream is of type REQUEST, it will be passed-through to the calculator. The receiving calculator will be in charge of deserializing it and interpreting all the content. Likewise, the output format RESPONSE delegate to the calculator creating a complete KServe response message to the client. That gives extra flexibility in the data format.
 
-```
+**Note:** For list of supported packet types and tags of `OpenVINOInferenceCalculator`, including graph examples, check documentation of [OpenVINO Model Server calculators](https://github.com/openvinotoolkit/mediapipe/blob/main/mediapipe/calculators/ovms/).
 
-{
-    "model_config_list": [...],
-    "mediapipe_config_list": [
-        {
-            "name":"mediaDummy",
-            "base_path":"/mediapipe/graphs/",
-            "graph_path":"graphdummyadapterfull.pbtxt",
-            "subconfig":"subconfig_dummy.json"
-    }
-    ]
-}
+### Side packets
+Side packets are special parameters which can be passed to the calculators at the beginning of the graph initialization. It can tune the behavior of the calculator like set the object detection threshold or number of objects to process.
+With KServe gRPC API you are also able to push side input packets into graph. They are to be passed as KServe request parameters. They can be of type `string`, `int64` or `boolean`.
```

Comment:
Shall we say how to access them "on the other side" or we don't want to get into details in this document?